audit-service/src/unit-tests/java/uk/gov/gchq/palisade/service/audit/web/SerDesHealthIndicatorTest.java

```diff
@@ -35,27 +39,20 @@ void tearDown() {
 
     @Test
     void testHealthUp() {
-        // Given there are no errors
-
-        // When getting the health
-        Health actual = healthIndicator.health();
-
-        // Then check the health is UP
-        assertThat(actual.getStatus()).as("Check the health status is UP").isEqualTo(Status.UP);
+        assertThat(healthIndicator.health().getStatus())
+            .as("Check health status is UP")
+            .isEqualTo(Status.UP);
     }
 
     @Test
     void testHealthDown() {
-        // Given the service has encountered at least 1 serialisation exception
+        // service has encountered at least 1 serialisation exception
         SerDesHealthIndicator.addSerDesExceptions(new Exception("This is a test"));
-
-        // When getting the health
-        Health actual = healthIndicator.health();
-
-        // Then check the health is DOWN
-        assertAll(
-                () -> assertThat(actual.getStatus()).as("Check the health status is DOWN").isEqualTo(Status.DOWN),
-                () -> assertThat(actual.getDetails()).as("Check the health details are not empty").isNotEmpty()
-        );
+        assertThat(healthIndicator.health().getStatus())
+            .as("Check health status as DOWN")
+            .isEqualTo(Status.DOWN);
+        assertThat(healthIndicator.health().getDetails())
+            .as("Check health status has some details")
+            .isNotEmpty();
```

Comment:
do we get any more details here? does the details contain some sort of body we can assert against? rather than just isNotEmpty?

Reply:
An attempt was made to check the body of the details but this was not working. I think that this is the only assertion that can be made as we only need to check an error was added but not necessarily the details of what was added.